Spruce up ephemeral workspace docs (#467)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/deletion.mdx

@include 'tfc-package-callouts/ephemeral-workspaces.mdx'
<!-- END: TFC:only name:pnp-callout -->

You can schedule a run to destroy all resources in a workspace at a specific

> date and time. This ephemeral workspace option reduces your infrastructure

Reviewer: ```suggestion
date and time. This ephemeral-workspace option reduces your infrastructure
```

> date and time. This ephemeral workspace option reduces your infrastructure

Reviewer 2: > This ephemeral workspace option

just for context: Ephemeral Workspaces is an entitlement, which can technically have many features – but right now the only feature under this entitlement is the auto destroy functionality. swapping out `option` -> `entitlement` may be more clear here that it's tied to pricing/subscription. I don't feel strongly about this though, just sprinkling in context.

> date and time. This ephemeral workspace option reduces your infrastructure

Author: ```suggestion
date and time. This ephemeral workspace entitlement reduces your infrastructure
```

spend by helping you automatically destroy resources that you only need
temporarily, such as development environments.

> If you perform a destroy operation in the workspace before the scheduled auto

Reviewer 2: This is my bad, the way it should have been written before is more like 

```This setting is automatically unset whenever Terraform Cloud performs an auto destroy run```

So technically not any destroy runs, only the automatic ones will unset this field. The main context here is that users may not anticipate that the setting will automatically be updated once completed, so we wanted to let them know in some way.

Author: thanks! i'll clarify

> If you perform a destroy operation in the workspace before the scheduled auto
> destroy, Terraform Cloud will cancel the scheduled run. You can re-enable it to
> schedule another future auto destroy run. 

Author: ```suggestion
After Terraform Cloud performs an auto destroy run, it unsets the `auto-destroy` flag on the workspace. If you continue using the workspace, you can schedule another future auto destroy run to remove any new resources. 
```

!> **Note:** Automatic destroy plans _do not_ prompt you for apply approval in the Terraform Cloud user interface. We recommend only using this setting for development environments.

You can schedule an auto destroy run using the Terraform Cloud web UI, or the [workspace API](/terraform/cloud-docs/api-docs/workspaces).

To schedule an auto destroy run in Terraform Cloud:
1. Navigate to the workspace's **Settings** > **Destruction and Deletion** page.
1. Under **Automatically destroy**, click **Set up auto destroy**.
1. Enter the desired date and time. Terraform Cloud defaults to your local time zone for scheduling and displays how long until the scheduled operation.
1. Click **Confirm auto destroy**.

To cancel a scheduled auto destroy run in Terraform Cloud:
1. Navigate to the workspace's **Settings** > **Destruction and Deletion** page.
1. Under **Automatically destroy**, click **Edit** next to your scheduled run's details.
1. Click **Remove**.

> You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 24 hours before an auto destroy run, and to report auto destroy run results.

Author: ```suggestion
You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto destroy run, and to report auto destroy run results.
```
